shenyu-spring-boot-starter-plugin-auth/shenyu-spring-boot-starter-pulgin-auth.iml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All .iml files should be ingored: https://stackoverflow.com/questions/14251624/intellij-idea-hide-iml-files

Reply:
ok